src/engraving/rendering/dev/measurelayout.cpp

```diff
@@ -851,6 +849,8 @@ void MeasureLayout::getNextMeasure(LayoutContext& ctx)
             // TODO? maybe we do need to process it here to make it possible to enable later
             //if (!segment.enabled())
             //      continue;
+            const Drumset* drumset
+                = staff->part()->instrument(segment.tick())->useDrumset() ? staff->part()->instrument(measure->tick())->drumset() : 0;
```

Comment:
Is it intentional that the expression at the left side of the `?` uses `segment` and at the right side uses `measure`?
```c++
  staff->part()->instrument(segment.tick())->useDrumset() 
? staff->part()->instrument(measure->tick())->drumset()
: …
```
(Also, while at it, it might be good to create a local variable for `staff->part()->instrument(segment.tick())` in order not to calculate that twice, and replace `0` with `nullptr`)